### PR TITLE
bump KEDA dep to 2.3.0

### PR DESCRIPTION
**Pull request by zroubalik**
Signed-off-by: Zbynek Roubalik <zroubali@redhat.com>

### CODE REVIEW

File: out.json [hunk 1]
```diff
@@ -0,0 +1,156 @@
+{
```
Comment (lionelvillard) [inline out.json:1]: seems like a file that shouldn't be there?

Comment (zroubalik) [reply out.json:1]: omg, thanks!

Comment (zroubalik) [reply out.json]: Fixed

**Pull request merged**